### DISCUSSION TITLE
Can't get standalone a320nx to appear

### DISCUSSION THREAD

**Discussion by Pawingnut45** [flybywiresim/aircraft#4406] [Help]
Hi, 

Ive downloaded the new version of the installer and have been trying to install the stable version, and it will not appear in the simulator aircraft menu.

I've uninstalled everything, reinstalled everything and restart many many times, and no stand alone folder, and no airplane. 

am i missing something

**Comment by TLSY2** [comment]
do you have the correct community folder for the mods for mfs2020? Maybe you installed in wrong place, if not work maybe try manual install from their website.

**Reply by Pawingnut45** [reply]
I do. I went into Dev mode and found the folders the games use, and install it there.  Tried both the community and the onestore folder with no success.

**Comment by pilotsteve88** [comment]
Install the mod either via the zip file or the installer to your community folder as normal.

The mod now is a **standalone** aircraft not a modification of the default a320n, to fly/use the mod you need to select 'Aircraft' from the menu when you load the sim and scroll along to the 'Flybywire mod', you'll have 2 a320n on the menu, the default msfs2020 version and fbw mod.

**Reply by wrightgulfstream** [reply]
I have it installed but only have details but no picture of aircraft.
When i start msfs 2020 theres no aircraft on the runway.
from
Peter.

**Comment by Pawingnut45** [comment]
I've now removed all my mods, put the fresh installer version of FBW in the community folder, One store folder, and still nothing...

Good thing the 5.4 version of the sim based neo works....

**Comment by Scotty4681** [comment]
I am having the same issue downloaded the 5.4 stable version and copied it to the community folder and it does not show up in the sim at all just the original 3 aircraft

**Reply by jachu1978** [reply]
> Mam ten sam problem, pobrałem stabilną wersję 5.4 i skopiowałem ją do folderu społeczności i nie pojawia się w simie w ogóle tylko oryginalne 3 samoloty

https://github.com/flybywiresim/a32nx/discussions/4246
